server/api/reports.py

```diff
@@ -686,25 +671,73 @@ def sampled_batch_rows(election: Election, jurisdiction: Jurisdiction = None):
     # We only support one contest for batch audits
     assert len(list(election.contests)) == 1
     contest = list(election.contests)[0]
+
     rows.append(
         [
             "Jurisdiction Name",
             "Batch Name",
             "Ticket Numbers",
             "Audited?",
-            "Audit Result",
+            "Audit Results",
+            "Reported Results",
+            "Discrepancy",
         ]
     )
     for batch in batches:
+        reported_results = {
+            choice.name: batch.jurisdiction.batch_tallies[batch.name][contest.id][
+                choice.id
+            ]
+            for choice in contest.choices
+        }
+
+        audit_results = {
+            choice.name: next(
+                (
+                    result.result
+                    for result in batch.results
+                    if result.contest_choice_id == choice.id
+                ),
+                0,
+            )
+            for choice in contest.choices
+        }
+
+        is_audited = len(batch.results) > 0
+
+        error = (
+            macro.compute_error(
+                {contest.name: reported_results},
+                {contest.name: audit_results},
+                # Construct the contest object by hand instead of using
```

Comment:
Would it make more sense to add another `from_db_contest` function or add a flag to the existing one that does this? Are we likely to have to do this again?

Reply:
After reading this comment, I ended up figuring out another way to do this that just uses `from_db_contest` instead